common/djangoapps/util/testing.py

```diff
@@ -1,9 +1,18 @@
 import sys
 
+from datetime import datetime
+from pytz import UTC
 from mock import patch
 
 from django.conf import settings
 from django.core.urlresolvers import clear_url_caches, resolve
+from openedx.core.djangoapps.course_groups.models import CourseUserGroupPartitionGroup
```

Comment:
Should have said that this is a nit. If it is difficult to move them then I'm okay with leaving them, since we have a number of other such erroneous imports.